test/jdk/java/foreign/TestUnsupportedLinker.java

```diff
@@ -23,6 +23,7 @@
 
 /*
  * @test
+ * @ignore architecture can not be overridden by setting os.arch - delete test
```

Comment:
FWIW, we introduce a property in CABI to override it directly in the JEP patch: https://github.com/openjdk/jdk/pull/13079/files#diff-1e9526318ae485495af012cbf0450693d77fd8241be62afe422a75304de5aed1R28

Reply:
I'll drop the changes to foreign to avoid conflicts with PR #13079.